2018-11-27-19:58:22,tpch_1g,postgresql,3,1.618
2018-11-27-19:58:49,tpch_1g,postgresql,1,3.032
2018-11-27-19:58:49,tpch_1g,postgresql,2,0.795
2018-11-27-19:58:49,tpch_1g,postgresql,3,0.776
2018-11-27-19:58:49,tpch_1g,postgresql,4,0.305
2018-11-27-19:58:49,tpch_1g,postgresql,5,0.355
2018-11-27-19:58:49,tpch_1g,postgresql,6,0.430
2018-11-27-19:58:49,tpch_1g,postgresql,7,0.477
2018-11-27-19:58:49,tpch_1g,postgresql,8,0.465
2018-11-27-19:58:49,tpch_1g,postgresql,9,3.054
2018-11-27-19:58:49,tpch_1g,postgresql,10,0.994
2018-11-27-19:58:49,tpch_1g,postgresql,11,0.154
2018-11-27-19:58:49,tpch_1g,postgresql,12,0.645
2018-11-27-19:58:49,tpch_1g,postgresql,13,1.311
2018-11-27-19:58:49,tpch_1g,postgresql,14,0.495
2018-11-27-19:58:49,tpch_1g,postgresql,15,1.105
2018-11-27-19:58:49,tpch_1g,postgresql,16,0.948
2018-11-27-19:58:49,tpch_1g,postgresql,17,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/17.sql:20: ERROR:  type "fixeddecimal" does not exist
LINE 12:    '0.2'::fixeddecimal * avg(l_quantity)
                   ^
0.008
2018-11-27-19:58:49,tpch_1g,postgresql,18,3.473
2018-11-27-19:58:49,tpch_1g,postgresql,19,0.637
2018-11-27-19:58:49,tpch_1g,postgresql,20,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/20.sql:40: ERROR:  type "fixeddecimal" does not exist
LINE 24:      0.5::fixeddecimal * sum(l_quantity)
                   ^
0.008
2018-11-27-19:58:49,tpch_1g,postgresql,21,0.926
2018-11-27-19:58:49,tpch_1g,postgresql,22,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/22.sql:40: ERROR:  type "fixeddecimal" does not exist
LINE 21:      c_acctbal > 0.00::fixeddecimal
                                ^
0.009
2018-11-27-20:31:09,tpch_1g,postgresql,1,2.861
2018-11-27-20:31:09,tpch_1g,postgresql,2,0.468
2018-11-27-20:31:09,tpch_1g,postgresql,3,0.535
2018-11-27-20:31:09,tpch_1g,postgresql,4,0.257
2018-11-27-20:31:09,tpch_1g,postgresql,5,0.557
2018-11-27-20:31:09,tpch_1g,postgresql,6,0.501
2018-11-27-20:31:09,tpch_1g,postgresql,7,0.645
2018-11-27-20:31:09,tpch_1g,postgresql,8,0.460
2018-11-27-20:31:09,tpch_1g,postgresql,9,1.760
2018-11-27-20:31:09,tpch_1g,postgresql,10,0.610
2018-11-27-20:31:09,tpch_1g,postgresql,11,0.163
2018-11-27-20:31:09,tpch_1g,postgresql,12,0.652
2018-11-27-20:31:09,tpch_1g,postgresql,13,1.901
2018-11-27-20:31:09,tpch_1g,postgresql,14,0.418
2018-11-27-20:31:09,tpch_1g,postgresql,15,1.589
2018-11-27-20:31:09,tpch_1g,postgresql,16,0.933
2018-11-27-20:31:09,tpch_1g,postgresql,17,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/17.sql:20: ERROR:  type "fixeddecimal" does not exist
LINE 12:    '0.2'::fixeddecimal * avg(l_quantity)
                   ^
0.008
2018-11-27-20:31:09,tpch_1g,postgresql,18,3.525
2018-11-27-20:31:09,tpch_1g,postgresql,19,0.071
2018-11-27-20:31:09,tpch_1g,postgresql,20,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/20.sql:40: ERROR:  type "fixeddecimal" does not exist
LINE 24:      0.5::fixeddecimal * sum(l_quantity)
                   ^
0.008
2018-11-27-20:31:09,tpch_1g,postgresql,21,0.924
2018-11-27-20:31:09,tpch_1g,postgresql,22,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/22.sql:40: ERROR:  type "fixeddecimal" does not exist
LINE 21:      c_acctbal > 0.00::fixeddecimal
                                ^
0.008
2018-11-27-20:35:02,tpch_1g,postgresql,1,2.897
2018-11-27-20:35:02,tpch_1g,postgresql,2,0.488
2018-11-27-20:35:02,tpch_1g,postgresql,3,0.526
2018-11-27-20:35:02,tpch_1g,postgresql,4,0.255
2018-11-27-20:35:02,tpch_1g,postgresql,5,0.551
2018-11-27-20:35:02,tpch_1g,postgresql,6,0.502
2018-11-27-20:35:02,tpch_1g,postgresql,7,0.640
2018-11-27-20:35:02,tpch_1g,postgresql,8,0.238
2018-11-27-20:35:02,tpch_1g,postgresql,9,1.757
2018-11-27-20:35:02,tpch_1g,postgresql,10,0.606
2018-11-27-20:35:02,tpch_1g,postgresql,11,0.165
2018-11-27-20:35:02,tpch_1g,postgresql,12,0.646
2018-11-27-20:35:02,tpch_1g,postgresql,13,1.854
2018-11-27-20:35:02,tpch_1g,postgresql,14,0.427
2018-11-27-20:35:02,tpch_1g,postgresql,15,1.592
2018-11-27-20:35:02,tpch_1g,postgresql,16,0.920
2018-11-27-20:35:02,tpch_1g,postgresql,17,1.068
2018-11-27-20:35:02,tpch_1g,postgresql,18,3.471
2018-11-27-20:35:02,tpch_1g,postgresql,19,0.070
2018-11-27-20:35:02,tpch_1g,postgresql,20,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/20.sql:40: ERROR:  type "fixeddecimal" does not exist
LINE 24:      0.5::fixeddecimal * sum(l_quantity)
                   ^
0.008
2018-11-27-20:35:02,tpch_1g,postgresql,21,0.924
2018-11-27-20:35:02,tpch_1g,postgresql,22,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/22.sql:40: ERROR:  type "fixeddecimal" does not exist
LINE 21:      c_acctbal > 0.00::fixeddecimal
                                ^
0.009
2018-11-27-20:37:02,tpch_1g,postgresql,1,2.884
2018-11-27-20:37:02,tpch_1g,postgresql,2,0.476
2018-11-27-20:37:02,tpch_1g,postgresql,3,0.522
2018-11-27-20:37:02,tpch_1g,postgresql,4,0.257
2018-11-27-20:37:02,tpch_1g,postgresql,5,0.551
2018-11-27-20:37:02,tpch_1g,postgresql,6,0.500
2018-11-27-20:37:02,tpch_1g,postgresql,7,0.643
2018-11-27-20:37:02,tpch_1g,postgresql,8,0.241
2018-11-27-20:37:02,tpch_1g,postgresql,9,1.739
2018-11-27-20:37:02,tpch_1g,postgresql,10,0.606
2018-11-27-20:37:02,tpch_1g,postgresql,11,0.163
2018-11-27-20:37:02,tpch_1g,postgresql,12,0.644
2018-11-27-20:37:02,tpch_1g,postgresql,13,1.893
2018-11-27-20:37:02,tpch_1g,postgresql,14,0.411
2018-11-27-20:37:02,tpch_1g,postgresql,15,1.553
2018-11-27-20:37:02,tpch_1g,postgresql,16,0.928
2018-11-27-20:37:02,tpch_1g,postgresql,17,1.072
2018-11-27-20:37:02,tpch_1g,postgresql,18,3.609
2018-11-27-20:37:02,tpch_1g,postgresql,19,0.070
2018-11-27-20:37:02,tpch_1g,postgresql,20,0.365
2018-11-27-20:37:02,tpch_1g,postgresql,21,0.935
2018-11-27-20:37:02,tpch_1g,postgresql,22,0.193
2018-11-28-14:54:39,tpch_1g,postgresql,1,2.903
2018-11-28-14:54:39,tpch_1g,postgresql,2,0.484
2018-11-28-14:54:39,tpch_1g,postgresql,3,0.528
2018-11-28-14:54:39,tpch_1g,postgresql,4,0.257
2018-11-28-14:54:39,tpch_1g,postgresql,5,0.551
2018-11-28-14:54:39,tpch_1g,postgresql,6,0.501
2018-11-28-14:54:39,tpch_1g,postgresql,7,0.642
2018-11-28-14:54:39,tpch_1g,postgresql,8,0.239
2018-11-28-14:54:39,tpch_1g,postgresql,9,1.737
2018-11-28-14:54:39,tpch_1g,postgresql,10,0.717
2018-11-28-14:54:39,tpch_1g,postgresql,11,0.191
2018-11-28-14:54:39,tpch_1g,postgresql,12,0.645
2018-11-28-14:54:39,tpch_1g,postgresql,13,1.968
2018-11-28-14:54:39,tpch_1g,postgresql,14,0.413
2018-11-28-14:54:39,tpch_1g,postgresql,15,1.552
2018-11-28-14:54:39,tpch_1g,postgresql,16,0.933
2018-11-28-14:54:39,tpch_1g,postgresql,17,1.091
2018-11-28-14:54:39,tpch_1g,postgresql,18,3.179
2018-11-28-14:54:39,tpch_1g,postgresql,19,0.068
2018-11-28-14:54:39,tpch_1g,postgresql,20,0.364
2018-11-28-14:54:39,tpch_1g,postgresql,21,0.925
2018-11-28-14:54:39,tpch_1g,postgresql,22,0.194
2018-11-28-14:55:03,tpch_3g,postgresql,1,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/1.sql:24: ERROR:  relation "lineitem" does not exist
LINE 13:  lineitem
          ^
0.090
2018-11-28-14:55:03,tpch_3g,postgresql,2,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/2.sql:47: ERROR:  relation "part" does not exist
LINE 11:  part,
          ^
0.012
2018-11-28-14:55:03,tpch_3g,postgresql,3,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/3.sql:26: ERROR:  relation "customer" does not exist
LINE 7:  customer,
         ^
0.011
2018-11-28-14:55:03,tpch_3g,postgresql,4,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/4.sql:24: ERROR:  relation "orders" does not exist
LINE 5:  orders
         ^
0.009
2018-11-28-14:55:03,tpch_3g,postgresql,5,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/5.sql:27: ERROR:  relation "customer" does not exist
LINE 5:  customer,
         ^
0.009
2018-11-28-14:55:03,tpch_3g,postgresql,6,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/6.sql:12: ERROR:  relation "lineitem" does not exist
LINE 4:  lineitem
         ^
0.008
2018-11-28-14:55:03,tpch_3g,postgresql,7,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/7.sql:42: ERROR:  relation "supplier" does not exist
LINE 14:    supplier,
            ^
0.008
2018-11-28-14:55:03,tpch_3g,postgresql,8,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/8.sql:40: ERROR:  relation "part" does not exist
LINE 14:    part,
            ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,9,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/9.sql:35: ERROR:  relation "part" does not exist
LINE 12:    part,
            ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,10,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/10.sql:35: ERROR:  relation "customer" does not exist
LINE 11:  customer,
          ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,11,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/11.sql:30: ERROR:  relation "partsupp" does not exist
LINE 5:  partsupp,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,12,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/12.sql:31: ERROR:  relation "orders" does not exist
LINE 16:  orders,
          ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,13,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/13.sql:23: ERROR:  relation "customer" does not exist
LINE 10:    customer left outer join orders on
            ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,14,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/14.sql:16: ERROR:  relation "lineitem" does not exist
LINE 8:  lineitem,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,15,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/15.sql:13: ERROR:  relation "lineitem" does not exist
LINE 6:   lineitem
          ^
psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/15.sql:34: ERROR:  relation "supplier" does not exist
LINE 8:  supplier,
         ^
psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/15.sql:36: ERROR:  view "revenue0" does not exist
0.008
2018-11-28-14:55:03,tpch_3g,postgresql,16,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/16.sql:33: ERROR:  relation "partsupp" does not exist
LINE 7:  partsupp,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,17,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/17.sql:20: ERROR:  relation "lineitem" does not exist
LINE 4:  lineitem,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,18,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/18.sql:36: ERROR:  relation "customer" does not exist
LINE 9:  customer,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,19,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/19.sql:38: ERROR:  relation "lineitem" does not exist
LINE 4:  lineitem,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,20,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/20.sql:40: ERROR:  relation "supplier" does not exist
LINE 5:  supplier,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,21,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/21.sql:43: ERROR:  relation "supplier" does not exist
LINE 5:  supplier,
         ^
0.007
2018-11-28-14:55:03,tpch_3g,postgresql,22,psql:/home/grid/eipi10/python-book/arsenal/tpch_test/postgresql/query/22.sql:40: ERROR:  relation "customer" does not exist
LINE 11:    customer
            ^
0.007
2018-11-28-15:19:13,tpch_3g,postgresql,1,8.379
2018-11-28-15:19:13,tpch_3g,postgresql,2,1.583
2018-11-28-15:19:13,tpch_3g,postgresql,3,1.849
2018-11-28-15:19:13,tpch_3g,postgresql,4,0.740
2018-11-28-15:19:13,tpch_3g,postgresql,5,1.898
2018-11-28-15:19:13,tpch_3g,postgresql,6,1.231
2018-11-28-15:19:13,tpch_3g,postgresql,7,2.021
2018-11-28-15:19:13,tpch_3g,postgresql,8,0.752
2018-11-28-15:19:13,tpch_3g,postgresql,9,5.628
2018-11-28-15:19:13,tpch_3g,postgresql,10,2.209
2018-11-28-15:19:13,tpch_3g,postgresql,11,0.789
2018-11-28-15:19:13,tpch_3g,postgresql,12,1.875
2018-11-28-15:19:13,tpch_3g,postgresql,13,8.528
2018-11-28-15:19:13,tpch_3g,postgresql,14,0.848
2018-11-28-15:19:13,tpch_3g,postgresql,15,4.232
2018-11-28-15:19:13,tpch_3g,postgresql,16,1.938
2018-11-28-15:19:13,tpch_3g,postgresql,17,3.429
2018-11-28-15:19:13,tpch_3g,postgresql,18,9.308
2018-11-28-15:19:13,tpch_3g,postgresql,19,0.190
2018-11-28-15:19:13,tpch_3g,postgresql,20,1.080
2018-11-28-15:19:13,tpch_3g,postgresql,21,4.095
2018-11-28-15:19:13,tpch_3g,postgresql,22,0.533
2018-11-28-15:22:28,tpch_3g,postgresql,1,8.267
2018-11-28-15:22:28,tpch_3g,postgresql,2,1.555
2018-11-28-15:22:28,tpch_3g,postgresql,3,1.831
2018-11-28-15:22:28,tpch_3g,postgresql,4,0.737
2018-11-28-15:22:28,tpch_3g,postgresql,5,1.879
2018-11-28-15:22:28,tpch_3g,postgresql,6,1.208
2018-11-28-15:22:28,tpch_3g,postgresql,7,1.969
2018-11-28-15:22:28,tpch_3g,postgresql,8,0.727
2018-11-28-15:22:28,tpch_3g,postgresql,9,5.558
2018-11-28-15:22:28,tpch_3g,postgresql,10,2.088
2018-11-28-15:22:28,tpch_3g,postgresql,11,0.782
2018-11-28-15:22:28,tpch_3g,postgresql,12,1.866
2018-11-28-15:22:28,tpch_3g,postgresql,13,8.435
2018-11-28-15:22:28,tpch_3g,postgresql,14,0.842
2018-11-28-15:22:28,tpch_3g,postgresql,15,4.098
2018-11-28-15:22:28,tpch_3g,postgresql,16,1.940
2018-11-28-15:22:28,tpch_3g,postgresql,17,3.376
2018-11-28-15:22:28,tpch_3g,postgresql,18,10.522
2018-11-28-15:22:28,tpch_3g,postgresql,19,0.161
2018-11-28-15:22:28,tpch_3g,postgresql,20,1.071
2018-11-28-15:22:28,tpch_3g,postgresql,21,4.123
2018-11-28-15:22:28,tpch_3g,postgresql,22,0.532
2018-11-28-15:30:57,tpch_3g,postgresql,1,8.345
2018-11-28-15:30:57,tpch_3g,postgresql,2,1.576
2018-11-28-15:30:57,tpch_3g,postgresql,3,1.814
2018-11-28-15:30:57,tpch_3g,postgresql,4,0.732
2018-11-28-15:30:57,tpch_3g,postgresql,5,1.875
2018-11-28-15:30:57,tpch_3g,postgresql,6,1.202
2018-11-28-15:30:57,tpch_3g,postgresql,7,1.979
2018-11-28-15:30:57,tpch_3g,postgresql,8,0.733
2018-11-28-15:30:57,tpch_3g,postgresql,9,5.593
2018-11-28-15:30:57,tpch_3g,postgresql,10,2.080
2018-11-28-15:30:57,tpch_3g,postgresql,11,0.794
2018-11-28-15:30:57,tpch_3g,postgresql,12,1.875
2018-11-28-15:30:57,tpch_3g,postgresql,13,8.372
2018-11-28-15:30:57,tpch_3g,postgresql,14,0.842
2018-11-28-15:30:57,tpch_3g,postgresql,15,4.093
2018-11-28-15:30:57,tpch_3g,postgresql,16,1.945
2018-11-28-15:30:57,tpch_3g,postgresql,17,3.381
2018-11-28-15:30:57,tpch_3g,postgresql,18,9.947
2018-11-28-15:30:57,tpch_3g,postgresql,19,0.160
2018-11-28-15:30:57,tpch_3g,postgresql,20,1.064
2018-11-28-15:30:57,tpch_3g,postgresql,21,4.104
2018-11-28-15:30:57,tpch_3g,postgresql,22,0.516
2018-11-28-15:35:16,tpch_3g,postgresql,1,8.384
2018-11-28-15:35:16,tpch_3g,postgresql,2,1.643
2018-11-28-15:35:16,tpch_3g,postgresql,3,1.841
2018-11-28-15:35:16,tpch_3g,postgresql,4,0.749
2018-11-28-15:35:16,tpch_3g,postgresql,5,1.938
2018-11-28-15:35:16,tpch_3g,postgresql,6,1.225
2018-11-28-15:35:16,tpch_3g,postgresql,7,2.046
2018-11-28-15:35:16,tpch_3g,postgresql,8,0.750
2018-11-28-15:35:16,tpch_3g,postgresql,9,5.707
2018-11-28-15:35:16,tpch_3g,postgresql,10,2.115
2018-11-28-15:35:16,tpch_3g,postgresql,11,0.822
2018-11-28-15:35:16,tpch_3g,postgresql,12,1.875
2018-11-28-15:35:16,tpch_3g,postgresql,13,8.615
2018-11-28-15:35:16,tpch_3g,postgresql,14,0.854
2018-11-28-15:35:16,tpch_3g,postgresql,15,4.135
2018-11-28-15:35:16,tpch_3g,postgresql,16,1.921
2018-11-28-15:35:16,tpch_3g,postgresql,17,3.475
2018-11-28-15:35:16,tpch_3g,postgresql,18,9.521
2018-11-28-15:35:16,tpch_3g,postgresql,19,0.163
2018-11-28-15:35:16,tpch_3g,postgresql,20,1.107
2018-11-28-15:35:16,tpch_3g,postgresql,21,4.172
2018-11-28-15:35:16,tpch_3g,postgresql,22,0.529
2018-11-28-15:51:44,tpch_3g,postgresql,1,8.393
2018-11-28-15:51:44,tpch_3g,postgresql,2,1.587
2018-11-28-15:51:44,tpch_3g,postgresql,3,1.848
2018-11-28-15:51:44,tpch_3g,postgresql,4,0.759
2018-11-28-15:51:44,tpch_3g,postgresql,5,1.945
2018-11-28-15:51:44,tpch_3g,postgresql,6,1.221
2018-11-28-15:51:44,tpch_3g,postgresql,7,2.047
2018-11-28-15:51:44,tpch_3g,postgresql,8,0.755
2018-11-28-15:51:44,tpch_3g,postgresql,9,5.730
2018-11-28-15:51:44,tpch_3g,postgresql,10,2.131
2018-11-28-15:51:44,tpch_3g,postgresql,11,0.825
2018-11-28-15:51:44,tpch_3g,postgresql,12,1.882
2018-11-28-15:51:44,tpch_3g,postgresql,13,8.654
2018-11-28-15:51:44,tpch_3g,postgresql,14,1.035
2018-11-28-15:51:44,tpch_3g,postgresql,15,4.130
2018-11-28-15:51:44,tpch_3g,postgresql,16,1.921
2018-11-28-15:51:44,tpch_3g,postgresql,17,3.465
2018-11-28-15:51:44,tpch_3g,postgresql,18,9.405
2018-11-28-15:51:44,tpch_3g,postgresql,19,0.165
2018-11-28-15:51:44,tpch_3g,postgresql,20,1.105
2018-11-28-15:51:44,tpch_3g,postgresql,21,4.168
2018-11-28-15:51:44,tpch_3g,postgresql,22,0.526
2018-11-28-16:29:19,tpch_3g,postgresql,1,8.326
2018-11-28-16:29:19,tpch_3g,postgresql,2,1.638
2018-11-28-16:29:19,tpch_3g,postgresql,3,1.858
2018-11-28-16:29:19,tpch_3g,postgresql,4,0.748
2018-11-28-16:29:19,tpch_3g,postgresql,5,1.919
2018-11-28-16:29:19,tpch_3g,postgresql,6,1.214
2018-11-28-16:29:19,tpch_3g,postgresql,7,2.028
2018-11-28-16:29:19,tpch_3g,postgresql,8,0.742
2018-11-28-16:29:19,tpch_3g,postgresql,9,5.658
2018-11-28-16:29:19,tpch_3g,postgresql,10,2.091
2018-11-28-16:29:19,tpch_3g,postgresql,11,0.838
2018-11-28-16:29:19,tpch_3g,postgresql,12,1.863
2018-11-28-16:29:19,tpch_3g,postgresql,13,8.175
2018-11-28-16:29:19,tpch_3g,postgresql,14,0.844
2018-11-28-16:29:19,tpch_3g,postgresql,15,4.099
2018-11-28-16:29:19,tpch_3g,postgresql,16,1.941
2018-11-28-16:29:19,tpch_3g,postgresql,17,3.427
2018-11-28-16:29:19,tpch_3g,postgresql,18,9.296
2018-11-28-16:29:19,tpch_3g,postgresql,19,0.164
2018-11-28-16:29:19,tpch_3g,postgresql,20,1.116
2018-11-28-16:29:19,tpch_3g,postgresql,21,4.120
2018-11-28-16:29:19,tpch_3g,postgresql,22,0.525
2018-11-28-21:03:13,tpch_1g,postgresql,1,2.917
2018-11-28-21:03:13,tpch_1g,postgresql,2,0.599
2018-11-28-21:03:13,tpch_1g,postgresql,3,0.534
2018-11-28-21:03:13,tpch_1g,postgresql,4,0.260
2018-11-28-21:03:13,tpch_1g,postgresql,5,0.561
2018-11-28-21:03:13,tpch_1g,postgresql,6,0.533
2018-11-28-21:03:13,tpch_1g,postgresql,7,0.651
2018-11-28-21:03:13,tpch_1g,postgresql,8,1.435
2018-11-28-21:03:13,tpch_1g,postgresql,9,2.490
2018-11-28-21:03:13,tpch_1g,postgresql,10,0.622
2018-11-28-21:03:13,tpch_1g,postgresql,11,0.246
2018-11-28-21:03:13,tpch_1g,postgresql,12,0.657
2018-11-28-21:03:13,tpch_1g,postgresql,13,1.833
2018-11-28-21:03:13,tpch_1g,postgresql,14,0.423
2018-11-28-21:03:13,tpch_1g,postgresql,15,1.575
2018-11-28-21:03:13,tpch_1g,postgresql,16,0.954
2018-11-28-21:03:13,tpch_1g,postgresql,17,1.227
2018-11-28-21:03:13,tpch_1g,postgresql,18,3.515
2018-11-28-21:03:13,tpch_1g,postgresql,19,0.189
2018-11-28-21:03:13,tpch_1g,postgresql,20,0.852
2018-11-28-21:03:13,tpch_1g,postgresql,21,0.968
2018-11-28-21:03:13,tpch_1g,postgresql,22,0.184
2018-11-29-16:53:23,tpch_1g,redshift,1,psql: could not connect to server: Connection timed out
	Is the server running on host "csswechat-test.cgqrfxjfcncp.cn-northwest-1.redshift.amazonaws.com.cn" (172.31.101.141) and accepting
	TCP/IP connections on port 5432?
127.337
2018-11-29-17:07:02,tpch_1g,redshift,1,7.767
2018-11-29-17:07:02,tpch_1g,redshift,2,34.878
2018-11-29-17:07:02,tpch_1g,redshift,3,13.958
2018-11-29-17:07:02,tpch_1g,redshift,4,psql:/home/xu6/tpch/postgresql/query/4.sql:24: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.030
2018-11-29-17:07:02,tpch_1g,redshift,5,23.520
2018-11-29-17:07:02,tpch_1g,redshift,6,5.534
2018-11-29-17:07:02,tpch_1g,redshift,7,26.366
2018-11-29-17:07:02,tpch_1g,redshift,8,18.158
2018-11-29-17:07:02,tpch_1g,redshift,9,14.789
2018-11-29-17:07:02,tpch_1g,redshift,10,15.290
2018-11-29-17:07:02,tpch_1g,redshift,11,12.069
2018-11-29-17:07:02,tpch_1g,redshift,12,10.585
2018-11-29-17:07:02,tpch_1g,redshift,13,14.609
2018-11-29-17:07:02,tpch_1g,redshift,14,11.117
2018-11-29-17:07:02,tpch_1g,redshift,15,18.618
2018-11-29-17:07:02,tpch_1g,redshift,16,17.255
2018-11-29-17:07:02,tpch_1g,redshift,17,14.358
2018-11-29-17:07:02,tpch_1g,redshift,18,20.596
2018-11-29-17:07:02,tpch_1g,redshift,19,6.980
2018-11-29-17:07:02,tpch_1g,redshift,20,psql:/home/xu6/tpch/postgresql/query/20.sql:40: ERROR:  This type of correlated subquery pattern is not supported due to internal error
6.958
2018-11-29-17:07:02,tpch_1g,redshift,21,44.235
2018-11-29-17:07:02,tpch_1g,redshift,22,15.410
2018-11-29-18:19:11,tpch_1g,redshift,1,0.022
2018-11-29-18:19:11,tpch_1g,redshift,2,0.018
2018-11-29-18:19:11,tpch_1g,redshift,3,0.016
2018-11-29-18:19:11,tpch_1g,redshift,4,psql:/home/xu6/tpch/postgresql/query/4.sql:24: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.028
2018-11-29-18:19:11,tpch_1g,redshift,5,0.015
2018-11-29-18:19:11,tpch_1g,redshift,6,0.015
2018-11-29-18:19:11,tpch_1g,redshift,7,0.016
2018-11-29-18:19:11,tpch_1g,redshift,8,0.016
2018-11-29-18:19:11,tpch_1g,redshift,9,0.016
2018-11-29-18:19:11,tpch_1g,redshift,10,0.016
2018-11-29-18:19:11,tpch_1g,redshift,11,0.016
2018-11-29-18:19:11,tpch_1g,redshift,12,0.015
2018-11-29-18:19:11,tpch_1g,redshift,13,0.015
2018-11-29-18:19:11,tpch_1g,redshift,14,0.015
2018-11-29-18:19:11,tpch_1g,redshift,15,1.477
2018-11-29-18:19:11,tpch_1g,redshift,16,0.080
2018-11-29-18:19:11,tpch_1g,redshift,17,0.015
2018-11-29-18:19:11,tpch_1g,redshift,18,0.015
2018-11-29-18:19:11,tpch_1g,redshift,19,0.016
2018-11-29-18:19:11,tpch_1g,redshift,20,psql:/home/xu6/tpch/postgresql/query/20.sql:40: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.096
2018-11-29-18:19:11,tpch_1g,redshift,21,0.017
2018-11-29-18:19:11,tpch_1g,redshift,22,0.016
2018-11-29-23:45:53,tpch_3g,redshift,1,5.500
2018-11-29-23:45:53,tpch_3g,redshift,2,4.512
2018-11-29-23:45:53,tpch_3g,redshift,3,2.120
2018-11-29-23:45:53,tpch_3g,redshift,4,psql:/home/xu6/tpch/postgresql/query/4.sql:24: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.030
2018-11-29-23:45:53,tpch_3g,redshift,5,2.174
2018-11-29-23:45:53,tpch_3g,redshift,6,0.781
2018-11-29-23:45:53,tpch_3g,redshift,7,3.937
2018-11-29-23:45:53,tpch_3g,redshift,8,3.412
2018-11-29-23:45:53,tpch_3g,redshift,9,5.113
2018-11-29-23:45:53,tpch_3g,redshift,10,8.647
2018-11-29-23:45:53,tpch_3g,redshift,11,4.333
2018-11-29-23:45:53,tpch_3g,redshift,12,3.108
2018-11-29-23:45:53,tpch_3g,redshift,13,2.970
2018-11-29-23:45:53,tpch_3g,redshift,14,0.598
2018-11-29-23:45:53,tpch_3g,redshift,15,3.325
2018-11-29-23:45:53,tpch_3g,redshift,16,2.836
2018-11-29-23:45:53,tpch_3g,redshift,17,1.605
2018-11-29-23:45:53,tpch_3g,redshift,18,9.761
2018-11-29-23:45:53,tpch_3g,redshift,19,4.019
2018-11-29-23:45:53,tpch_3g,redshift,20,psql:/home/xu6/tpch/postgresql/query/20.sql:40: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.147
2018-11-29-23:45:53,tpch_3g,redshift,21,11.173
2018-11-29-23:45:53,tpch_3g,redshift,22,0.728
2018-11-29-23:48:07,tpch_3g,redshift,1,0.021
2018-11-29-23:48:07,tpch_3g,redshift,2,0.017
2018-11-29-23:48:07,tpch_3g,redshift,3,0.015
2018-11-29-23:48:07,tpch_3g,redshift,4,psql:/home/xu6/tpch/postgresql/query/4.sql:24: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.028
2018-11-29-23:48:07,tpch_3g,redshift,5,0.016
2018-11-29-23:48:07,tpch_3g,redshift,6,0.015
2018-11-29-23:48:07,tpch_3g,redshift,7,0.015
2018-11-29-23:48:07,tpch_3g,redshift,8,0.016
2018-11-29-23:48:07,tpch_3g,redshift,9,0.016
2018-11-29-23:48:07,tpch_3g,redshift,10,0.016
2018-11-29-23:48:07,tpch_3g,redshift,11,0.015
2018-11-29-23:48:07,tpch_3g,redshift,12,0.015
2018-11-29-23:48:07,tpch_3g,redshift,13,0.015
2018-11-29-23:48:07,tpch_3g,redshift,14,0.015
2018-11-29-23:48:07,tpch_3g,redshift,15,1.214
2018-11-29-23:48:07,tpch_3g,redshift,16,0.821
2018-11-29-23:48:07,tpch_3g,redshift,17,0.016
2018-11-29-23:48:07,tpch_3g,redshift,18,0.016
2018-11-29-23:48:07,tpch_3g,redshift,19,0.016
2018-11-29-23:48:07,tpch_3g,redshift,20,psql:/home/xu6/tpch/postgresql/query/20.sql:40: ERROR:  This type of correlated subquery pattern is not supported due to internal error
0.146
2018-11-29-23:48:07,tpch_3g,redshift,21,0.017
2018-11-29-23:48:07,tpch_3g,redshift,22,0.015
